ReSharper.FSharp/src/FSharp.Fantomas.Host/Fantomas.Host.win.runtimeconfig.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "runtimeOptions": {
+    "tfm": "net6",
+    "rollForward": "Major",
+    "framework": {
+      "name": "Microsoft.WindowsDesktop.App",
```

Comment:
_(Minor)_ I wonder: why does it require `Microsoft.WindowsDesktop.App` on Windows?

Reply:
RD-protocol we use requires System.Windows.Threading.Dispatcher